recipe/build.sh

```diff
@@ -1,40 +1,63 @@
 #!/bin/bash
 
-mkdir build
-cd build
+SHORT_OS_STR=$(uname -s)
 
-if [ "$(expr substr $(uname -s) 1 5)" == "Linux" ]; then
-  DYNAMIC_EXT="so"
-  OPENMP="-DWITH_OPENMP=1"
+if [ "${SHORT_OS_STR:0:5}" == "Linux" ]; then
+    DYNAMIC_EXT="so"
+    OPENMP="-DWITH_OPENMP=1"
+    # There's a bug with CMake at the moment whereby it can't download
+    # using HTTPS - so we use curl to download the IPP library
+    mkdir -p $SRC_DIR/3rdparty/ippicv/downloads/linux-808b791a6eac9ed78d32a7666804320e
```

Comment:
Good point. Added an issue ( https://github.com/conda-forge/cmake-feedstock/issues/7 ) on the CMake feedstock.